resources/kcp/charts/kyma-metrics-collector/templates/service-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+{{ include "kyma-metrics-collector.labels" . | indent 4 }}
+  name: {{ include "kyma-metrics-collector.fullname" . }}
+  namespace: kcp-system
```

Comment:
Let's not explicitly use namespace. Helm will install this resource in the namespace where the chart is installed.

Reply:
done